### ISSUE TITLE
Runner instances are incorrectly detected as orphaned and terminated

### ISSUE THREAD

**Issue by gertjanmaas** [philips-labs/terraform-aws-github-runner#167]
## Summary

When there are over 30 runners in a repo/organization the scale down lambda thinks new runners are orphans and will terminate them even while running a build.

## Steps to reproduce

1. Add 30 runners to your repository or organisation. These can be offline.
2. Trigger a new workflow run to generate a new instance via this project
3. Wait the configured time (`minimum_running_time_in_minutes` option or 5 minutes by default)
4. Cloudwatch logs on the scale down function shows that the newly created instance is an orphan

## What is the current *bug* behavior?

Runners get terminated while they should not be deleted.

## What is the expected *correct* behavior?

These runners should not be terminated in this scenario.

## Relevant logs and/or screenshots

```
2020-08-26T09:30:06.050Z 7d6440bb-27ba-4d17-ba26-3edc285b88c1 INFO Runner 'i-0d90f0e61ef64b847' is orphan, and will be removed.
2020-08-26T09:30:06.272Z 7d6440bb-27ba-4d17-ba26-3edc285b88c1 DEBUG Runner terminated.i-0d90f0e61ef64b847

```

## Possible fixes

In `modules/runners/lambdas/runners/src/scale-runners/scale-down.ts` in the `scaleDown` function the following code is used to retrieve registered runners.
```
 const registered = enableOrgLevel
      ? await githubAppClient.actions.listSelfHostedRunnersForOrg({
          org: repo.repoOwner,
        })
      : await githubAppClient.actions.listSelfHostedRunnersForRepo({
          owner: repo.repoOwner,
          repo: repo.repoName,
        });
```

This API is paginated and by default returns the first 30 runners. The page size can be upped to 100 runners, but to be sure we should get all the runners.

**Comment by npalm** [comment]
At least that would be a quick fix.

**Comment by HenryNguyen5** [comment]
Getting bitten by this too. I'll work on a fix this week if others arent able to pick it up

**Comment by npalm** [comment]
@gertjanmaas did you do anything on this issue already?

**Comment by gertjanmaas** [comment]
No, haven't had the time yet. I'm okay with anyone else picking this up. I think the issue describes the problem well enough to get started.

**Comment by gertjanmaas** [comment]
Fixed by #202
